src/models/form/dissolutionApproval.model.ts

```diff
@@ -1,10 +1,10 @@
 import OfficerType from 'app/models/dto/officerType.enum'
 
 export default interface DissolutionApprovalModel {
-  companyName: string,
-  companyNumber: string,
-  applicant: string,
+  companyName: string
+  companyNumber: string
+  applicant: string
   officerType: OfficerType
-  onBehalfName?: string,
+  onBehalfName?: string
   date: string
 }
```

Comment:
Should we add an extra line at the bottom of this file?

Reply:
Done - also added rule to tslint so had to fix all violations in the app

Reply:
That's good! :D